README.md

```diff
@@ -1,15 +1,12 @@
+# Gatsbyn Suomenkielinen Käännös
 
-# Gatsby Finnish Translation
+Tämä repo sisältää Gatsbyn suomenkielisen käännöksen.
 
-This repo contains the Finnish translation for Gatsby.
+Siirry katsomaan [tänne](https://github.com/gatsbyjs/gatsby-fi/issues/1) käännöksen etenemistä,
+jotta voit valita sivusi ja varmistaa, että kukaan ei ole aloittanut kääntämistä.
 
-Please refer to the [Translation Progress Issue](https://github.com/gatsbyjs/gatsby-fi/issues/1) to start translating!
-
-Useful Links:
+Hyödylliset linkit:
 
 * [Style Guide](/style-guide.md)
```

Comment:
Should `Style Guide` be translated? If yes, it would be `tyyliohje` probably?

Reply:
I've compared translations and many use `Style Guide`. It's up to you 😃.

Reply:
Sounds good :) Let's use style guide for that